### PR TITLE
Adjust diagnostic configuration files (DCOS_OSS-5662)

### PR DESCRIPTION
**Pull request by OleksandrBielov**
Adjust the code and change the location of the configuration files of the dcos-diagnostics package in order to deploy properly together with the winpanda

### CODE REVIEW

File: vendor/github.com/dcos/dcos-go/dcos/constants.go [hunk 1]
```diff
@@ -20,7 +20,8 @@ const (
 func GetFileDetectIPLocation() string {
 	switch runtime.GOOS {
 	case "windows":
-		return "/opt/mesosphere/bin/detect_ip.ps1"
+        return "c:\\dcos\\opt\\bin\\detect_ip.ps1"
```
Comment (janisz) [inline vendor/github.com/dcos/dcos-go/dcos/constants.go]: Can you update dcos-go to contain this update https://github.com/dcos/dcos-diagnostics/blob/5856695d58b817e07f590345abc813a56e3a9b2c/go.mod#L10

Comment (janisz) [reply vendor/github.com/dcos/dcos-go/dcos/constants.go]: https://github.com/dcos/dcos-go/blob/3b86d9c7fac36be9d16fc1d22e5a5dc54738aac0/dcos/constants.go#L23

**Comment by makkes** [comment]
@OleksandrBielov Are you still working on this?

**Comment by janisz** [comment]
https://github.com/dcos/dcos-go/pull/70

**Comment by janisz** [comment]
Closing due to lack of activity

**Pull request closed without merging**